libcore/FileUtils.vala

```diff
@@ -828,7 +828,7 @@ namespace Files.FileUtils {
             var parent = drop_file.get_parent ();
 
             if (parent != null && parent.equal (target_location)) {
-                valid_actions &= Gdk.DragAction.LINK; // Only LINK is valid
+                valid_actions = Gdk.DragAction.DEFAULT; // Ignore (GitHub bug #1759)
```

Comment:
According to docs:
> [DEFAULT](https://valadoc.org/gdk-3.0/Gdk.DragAction.DEFAULT.html) - Means nothing, and should not be used.

Also I'm getting the replace dialog here now for files which seems even less desirable than the symlink behavior

Reply:
Not sure what to use for "Do Nothing" then, I guess use `0`?  I'll  just drop this and revisit in Gtk4.